dataproxy/vercel-cli/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -eu
+
+yarn
+
+yarn -s vercel --token=$VERCEL_TOKEN --build-env PRISMA_GENERATE_DATAPROXY="true" --env DATAPROXY_VERCEL_CLI_DB_URL="$DATAPROXY_VERCEL_CLI_DB_URL" --prod --scope=$VERCEL_ORG_ID --confirm --force 1> deployment-url.txt
```

Comment:
We are only using the env var in this test setup, not `--data-proxy` I guess?

Edit: Oh yeah, obviously - we added this env var because the other thing did not work :D